pkgs/by-name/eu/eudic/package.nix

```
@@ -71,11 +70,16 @@ stdenv.mkDerivation (finalAttrs: {
     mkdir -p $out
     cp -r usr/* $out/
 
-    makeWrapper $out/share/eusoft-eudic/eudic $out/bin/eudic
+    makeWrapper $out/share/eusoft-eudic/eudic $out/bin/eudic \
+      --inherit-argv0
 
     runHook postInstall
   '';
 
+  postFixup = ''
+    sed -i "s:/usr/share/eusoft-eudic/AppRun:$out/bin/eudic:" $out/share/applications/eusoft-eudic.desktop
```

```suggestion
    substituteInPlace $out/share/applications/eusoft-eudic.desktop \
      --replace-fail '/usr/share/eusoft-eudic/AppRun' 'eudic'
```

https://github.com/NixOS/nixpkgs/issues/308324

---

Fixed and tested. Thanks for letting me know!